intellij-plugin-structure/build.gradle.kts

```diff
@@ -0,0 +1,232 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+  `maven-publish`
+  signing
+  id("org.jetbrains.kotlin.jvm") version "1.8.10"
+  id("io.github.gradle-nexus.publish-plugin") version "1.3.0"
+}
+
+val kotlin_version = "1.8.20"
+
+var intellijPluginStructureVersion = "dev"
+if (project.hasProperty("structureVersion")) {
+  val structureVersion = project.properties["structureVersion"].toString()
+  intellijPluginStructureVersion = structureVersion
+}
+
+allprojects {
+  group = "org.jetbrains.intellij.plugins"
+  version = intellijPluginStructureVersion
+
+  apply {
+    plugin("java")
+    plugin("kotlin")
+  }
+
+  java {
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
+  }
+
+  repositories {
+    maven {
+      url = uri("https://repo.maven.apache.org/maven2")
+    }
+    maven {
+      url = uri("https://www.jetbrains.com/intellij-repository/releases")
+    }
+    maven {
+      url = uri("https://packages.jetbrains.team/maven/p/ij/intellij-dependencies")
+    }
+  }
+  dependencies {
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version")
+    implementation("com.fasterxml.jackson.module:jackson-module-kotlin:2.14.2")
+    implementation("org.jetbrains.kotlin:kotlin-reflect:$kotlin_version")
+  }
+
+  val sourcesJar by tasks.registering(Jar::class) {
+    dependsOn("classes")
+    archiveClassifier.set("sources")
+    from(sourceSets.main.get().allSource)
+  }
+
+  val javadocJar by tasks.registering(Jar::class) {
+    val javadoc = tasks.javadoc
+    dependsOn(javadoc)
+    archiveClassifier.set("javadoc")
+    from(javadoc)
+  }
+
+  artifacts {
+    archives(sourcesJar)
+    archives(javadocJar)
+  }
+
+  tasks.withType<KotlinCompile> {
+    kotlinOptions {
+      jvmTarget = "11"
+      apiVersion = "1.4"
```

Comment:
Maybe try to update to 1.8?

Reply:
But probably in the next branch

Reply:
I'll leave that to the 1.4 for now. Version update will be in the subsequent branch.